tests/tag.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "tag" {
+  cid=$(buildah from --pull=false --signature-policy ${TESTSDIR}/policy.json scratch)
+  run buildah commit "$cid" scratch-image
+  [ "$status" -eq 0 ]
+  run buildah inspect --type image tagged-image
+  [ "$status" -ne 0 ]
+  run buildah tag scratch-image tagged-image tagged-also-image
+  [ "$status" -eq 0 ]
+  run buildah inspect --type image tagged-image
```

Comment:
IDK why, but 3 has always been the magical painful number for me.  If you could add a test with three tags added and tested for I'd feel  better.  Not required, but appreciated if you do.